math/vecops/inc/ROOT/TVec.hxx

```diff
@@ -214,23 +215,30 @@ private:
    }
 
 public:
-   // ctors
-   TVec() = default;
-   TVec(const TVec<T> &) = default;
-   TVec(TVec<T> &&) = default;
-   template <class InputIt>
-   TVec(InputIt first, InputIt last) : fData(first, last)
-   {
-   }
-   TVec(size_type count, const T &value) : fData(count, value) {}
+   // Note: We must define - rather than just declare - all constructors and
+   // operators here so that extern templates have all needed symbols. That is,
+   // we cannot use something like TVec(...) = default; since it only declares.
```

Comment:
That statement is incorrect; `= default` will work - once I fix the bug in cling! Working on it...

Reply:
Hi @Axel-Naumann, Ok, thanks for letting us know. If cling works with the `= default;` way of declaring the constructors, then I will drop this commit before merging.